tests/UI/campaigns/functional/BO/15_header/01_leftMenu.ts

```diff
@@ -1,25 +1,22 @@
 // Import utils
 import helper from '@utils/helpers';
+import loginCommon from '@commonTests/BO/loginBO';
 
 // Import test context
 import testContext from '@utils/testContext';
 
-require('module-alias/register');
-
-const {expect} = require('chai');
-
-// Import utils
-const loginCommon = require('@commonTests/BO/loginBO');
-
 // Import pages
-const dashboardPage = require('@pages/BO/dashboard');
+import dashboardPage from '@pages/BO/dashboard';
 
-const baseContext = 'functional_BO_header_leftMenu';
+import {expect} from 'chai';
+import {BrowserContext, Page} from 'playwright';
```

Comment:
we could only import types with 
```suggestion
import type {BrowserContext, Page} from 'playwright';
```

Reply:
Fixed